xplique/concepts/__init__.py

```diff
@@ -1,6 +1,12 @@
-"""
-Concept based methods
-"""
-
-from .cav import Cav
-from .tcav import Tcav
+"""
+Concept based methods
+"""
+
+from .cav import Cav
+from .tcav import Tcav
+from .craft import BaseCraft, DisplayImportancesOrder
```

Comment:
Do we really want to have BaseCraft here ? I mean is there any reason to want the user to import it directly from `concept`?

Reply:
- For `retrain`, it was to be able to regenerate the checkoints easily ; but ok I removed that.
- For the BaseCraft, it's because it is imported in both CraftTf and CraftTorch

Reply:
done